Core/src/main/java/com/paypal/android/core/AuthHandler.kt

```diff
@@ -0,0 +1,18 @@
+package com.paypal.android.core
+
+class AuthHandler private constructor(private val authHeaderValue: String) {
+
+    fun getAuthHeader() = authHeaderValue
+
+    companion object {
+
+        fun fromClientId(clientId: String): AuthHandler {
```

Comment:
Do we need this? We should only need a client token now.

Reply:
true